modules/exploits/unix/http/pfsense_config_data_exec.rb

```diff
@@ -0,0 +1,174 @@
+class MetasploitModule < Msf::Exploit::Remote
+  Rank = ExcellentRanking
+
+  include Msf::Exploit::Remote::HttpClient
+  include Msf::Exploit::CmdStager
+  include Msf::Exploit::FileDropper
+  prepend Msf::Exploit::Remote::AutoCheck
+
+  def initialize(info = {})
+    super(
+      update_info(
+        info,
+        'Name' => 'pfSense Restore RRD Data Command Injection',
+        'Description' => %q{
+          This module exploits an OS Command Injection vulnerability in the pfSense
+          Config Module (CVE-2023-27253). The vulnerability affects versions <= 2.7.0

```

Comment:
In https://github.com/rapid7/metasploit-framework/pull/17819/files#r1153744240 you mention that 2.7.0 is not yet released and will contain the patch. As such, presumably this should be:

```suggestion
          Config Module (CVE-2023-27253). The vulnerability affects versions < 2.7.0

```

or:

```suggestion
          Config Module (CVE-2023-27253). The vulnerability affects versions <= 2.6.0

```

Reply:
Yes, the vulnerability appears in version 2.6.0-RELEASE and lower. I'm making the change, thank you for the suggestion!